dataflux_pytorch/tests/test_dataflux_lightning_checkpoint.py

```diff
@@ -79,15 +78,6 @@ def test_empty_bucket_name(self):
             return
         self.fail("Empty bucket name expected to fail when parsed")
 
-    def test_invalid_bucket_name(self):
```

Comment:
This test compares the parsed bucket_name with one passed in init function. Previously there was a check in _parsce_gcs_path which would fail if both the bucket names were different. I removed that check and hence removed the corresponding test case as well.